data/remnant missions.txt

```diff
@@ -1252,3 +1252,214 @@ mission "Remnant: Expanded Horizons Quarg 3"
 			`	"Our historical records just have a few pictures and videos of Quarg ships, along with a basic scan that doesn't provide more than the general outline of the ship. These scans," he says while tapping the databanks, "actually picked up detailed structural layouts and even identified a number of key components. It isn't nearly enough information to be able to duplicate them, but it is enough that we can start trying to figure out how they work.`
 			`	Upon arriving at <planet> they quickly thank you, hand you <payment>, and run back to their offices to finish prepping their material to publish and distribute.`
 
+
+
+mission "Remnant: Broken Jump Drive 1"
+	name "Jump Drive (Broken)"
+	minor
+	invisible
+	landing
+	to offer
+		has "remnant blood test pure"
+	to complete
+		has "Jump Drive (Broken)"
+	source
+		attributes "remnant"
+	destination 
```

Comment:
I don't think you need to specify the destination at all here - the game will use the same `Planet` the mission offers on. (For boarding missions, you must specify it.)